contracts/liquidation/priceOracle/chainlink/ChainLinkPriceOracle.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+import {IPriceOracle} from '../../../interfaces/IPriceOracle.sol';
+import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
+import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+
+interface IChainLinkAggregatorProxy {
```

Comment:
If possible, pls add reference from chain-link repo

Reply:
Added link to interface from github